lib/tools/te_enum.h

```diff
@@ -296,6 +296,40 @@ extern int te_enum_translate(const te_enum_trn trn[],
                              int value, te_bool reverse,
                              int unknown_val);
 
+/**
+ * A mapping between a one one-bit mask from one bitmask and its
+ * corresponding one-bit mask in another bitmask.
+ *
+ * An array of mappings should end with TE_ENUM_BITMASK_CONV_END.
```

Comment:
`#TE_ENUM_BITMASK_CONV_END` or `::TE_ENUM_BITMASK_CONV_END` (see https://www.doxygen.nl/manual/autolink.html); if I've ever told somebody otherwise, I was wrong